### ISSUE TITLE
PgSQL执行某些命令提示“错误：无效的“UTF-8”编码字节顺序”

### ISSUE THREAD

**Issue by YS11i** [DeEpinGh0st/MDUT-Extend-Release#5]
```
org.postgresql.util.PSQLException: 错误: 无效的 "UTF8" 编码字节顺序: 0xc5 0xe4
  在位置：COPY cmd_exec, line 2
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2553)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2285)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:323)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:481)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:401)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:164)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:130)
	at Dao.PostgreSqlDao.cveEval(PostgreSqlDao.java:260)
	at Dao.PostgreSqlDao.eval(PostgreSqlDao.java:295)
	at Controller.PostgreSqlController.lambda$postgreSqlEval$9(PostgreSqlController.java:191)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by YS11i** [comment]
执行whoami正常，执行其他的ipconfig、dir等命令会提示上述错误

> ```
> org.postgresql.util.PSQLException: 错误: 无效的 "UTF8" 编码字节顺序: 0xc5 0xe4
>   在位置：COPY cmd_exec, line 2
> 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2553)
> 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2285)
> 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:323)
> 	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:481)
> 	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:401)
> 	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:164)
> 	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:130)
> 	at Dao.PostgreSqlDao.cveEval(PostgreSqlDao.java:260)
> 	at Dao.PostgreSqlDao.eval(PostgreSqlDao.java:295)
> 	at Controller.PostgreSqlController.lambda$postgreSqlEval$9(PostgreSqlController.java:191)
> 	at java.lang.Thread.run(Thread.java:748)
> ```

PostgreSql 版本：PostgreSQL 16.3
执行whoami正常，执行其他的ipconfig、dir等命令会提示上述错误，初步怀疑是windows的一些字符导致

**Comment by DeEpinGh0st** [comment]
发现这个问题了，正在修复

**Comment by DeEpinGh0st** [comment]
This issue has been resolved and will be closed by [Release v1.1.0](https://github.com/DeEpinGh0st/MDUT-Extend-Release/releases/tag/v1.1.0)

**Comment by DeEpinGh0st** [comment]
This issue has been resolved and will be closed by [Release v1.1.0](https://github.com/DeEpinGh0st/MDUT-Extend-Release/releases/tag/v1.1.0).
